app/test_split_integer.py

```diff
@@ -2,20 +2,28 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17,\
+        f"Sum of parts = {sum(split_integer(17, 4))}, "\
+        f"but it should be equal to 17"
```

Comment:
You don't need f-strings in cases, where you don't use links to objects, also remove this endslash.